src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import cautionSvg from '../../svgs/icon-caution-fill.svg';
+import Icon from '../icon/icon.jsx';
+import styles from './custom-field-input-text.css';
+
+function getRootClassName(className, error, disabled) {
+  const resolvedClassName = className || styles['custom-field-input-text'];
```

Comment:
This is a good time to explore the benefits of using a default property! Instead of using in-band logic we can leverage:

```
CustomFieldInputText.defaultProps = {
  className: styles['custom-field-input-text'],
};
```

Same thing but slicker! It _even_ is documented in our MDS Site!